environment.devenv.yml

```diff
@@ -1,8 +1,14 @@
 name: serialchemy
 
-includes:
-  - {{ root }}/../eden/sqlalchemy.devenv.yml
-
+dependencies:
```

Comment:
It is for general use.

Reply:
I don't think anyone would use DevEnv to develop this in isolation. We may get rid of it at all since in our env this lib will be used together with flask-restalchemy and  tornado-whaterver.

Reply:
You mean to develop it locally one would use a virtual env? That's a fine approach, but we also might want to accommodate the situations where you want to work with this repository in "develop" mode by checking out from sources and building it together with your application in the CI.